uint8 DEVICE_STATUS # UINT8 Length 1 byte ro
uint8 EJECTOR_STATUS # UINT8 Length 16 bytes ro
uint8 SUPPLY_VOLTAGE
uint16 LEAKAGE_RATE # UINT16 Length 16 x 2 ro
uint16 FREE_FLOW_VACUUM # UINT16 Length 16 x 2 ro
uint16 MAX_VACUUM_REACHED # UINT16 Length 16 x 2 ro
uint16 SYSTEM_VACUUM # UINT16 Length 16 x 2 ro

